.github/workflows/publish-dev-assets.yml

```diff
@@ -37,6 +37,10 @@ jobs:
           az version
           # Key Vault: 
           az account get-access-token --scope https://vault.azure.net/.default --output none
+      - name: Prepare notation certificate
+        run: |
+          mkdir -p truststore/x509/ca/ratify-verify
```

Comment:
wondering if its possible to extract common commands into sub workflows. Similar to https://github.com/ratify-project/ratify/blob/dev/.github/workflows/build-pr.yml#L15

Reply:
Yes I can do this however does it make sense given it's one step?

Reply:
ah, got it. If we have more shrable steps, we could move them out in the future.